curvefs/proto/mds.proto

```diff
@@ -82,6 +82,7 @@ message FsInfo {
     required common.FSType fsType = 9;
     required FsDetail detail = 10;
     required bool enableSumInDir = 11;
+    required string owner = 12;
```

Comment:
new fields should be `optional`, because when you have an older version client which doesn't have this fields and newer version mds, then client RPC to MDS will always fail

Reply:
if we have a stable version, we need to consider the compatibility, but now all are under development.
owner and capacity are all required for resource limitation and fs management.